src/Routes/Signatures/Details.js

```diff
@@ -88,14 +90,15 @@ const Details = () => {
                             <GridItem xl2={isCodeEditorEnabled && 6 || 2} xl={isCodeEditorEnabled && 6 || 3} md={6} xs={12}>
                                 {detailBlock(intl.formatMessage(messages.ruleIntendedUsage), sigMetadata?.usage)}
                             </GridItem>
-                            <GridItem xl2={isCodeEditorEnabled && 6 || 7} xl={isCodeEditorEnabled && 6 || 3} md={6} xs={12}>
-                                {sigMetadata?.xfti_reference &&
-                                    detailBlock(intl.formatMessage(messages.ruleAuthorReference), referenceLink(sigMetadata.xfti_reference))}
-                                {sigMetadata?.reference &&
-                                    detailBlock(intl.formatMessage(messages.ruleBackgroundReference), referenceLink(sigMetadata.reference))}
-                                {(!sigMetadata?.xfti_reference && !sigMetadata?.reference) &&
-                                    detailBlock(intl.formatMessage(messages.ruleReference), dataNotAvailable)}
-                            </GridItem>
+                            {xfti_references.length > 0 &&
```

Comment:
I personally like using the ternary operator to display conditionally as it explicitly states both cases and performs well. But this is an opinionated topic :).
`xfti_references.length > 0 ? referenceGridItem() : null`

Reply:
Thanks for the review @mkholjuraev .  Do you think it might be better to use a Dropdown component for the author reference links and background reference links, so they take up the same amount of space, no matter how many links there are?

Reply:
I think dropdown is more suitable for a form. For just displaying plain info, `[Expandable section](https://www.patternfly.org/v4/components/expandable-section)` should be appropriate.